nx/lib/nx/lin_alg/svd.ex

```diff
@@ -0,0 +1,259 @@
+defmodule Nx.LinAlg.SVD do
+  import Nx.Defn
+  @default_eps 1.1920929e-07
+
+  defn svd(input_tensor, opts \\ []) do
+    opts = keyword!(opts, max_iter: 100, eps: @default_eps)
```

Comment:
Use `Keyword.fetch!` here. The default should be set in a single place inside `Nx.LinAlg.svd`.